doc/api/errors.md

```diff
@@ -2890,6 +2890,17 @@ The WASI instance has already started.
 
 The WASI instance has not been started.
 
+<a id="ERR_WEBASSEMBLY_RESPONSE"></a>
+
+### `ERR_WEBASSEMBLY_RESPONSE`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+The `Response` that has been passed to `WebAssembly.compileStreaming` or to
+`WebAssembly.instantiateStreaming` is not a valid WebAssembly response.
```

Comment:
Should we link to a MDN resource that explains what is a valid WASM response?

Reply:
I didn't find a relevant MDN section. We could link to the spec but it contains irrelevant information (for example, this PR does not implement CORS) so it might not be a good reference either.

Reply:
Yeah, it's probably fine as is, we can leave it to the error message to explain what is the actual problem.